src/components/BlackKey.js

```diff
@@ -0,0 +1,21 @@
+import React, {useState} from 'react'
```

Comment:
Compartmentalization of code makes it certainly easier to go through as well makes it modular. Well done.